.github/workflows/tests.yml

```diff
@@ -134,14 +134,17 @@ jobs:
         run: |
           mkdir -p crdb
           pushd crdb
-          wget -qO- https://binaries.cockroachdb.com/cockroach-v2.1.0.linux-amd64.tgz | tar -xz
-          sudo cp -i cockroach-v2.1.0.linux-amd64/cockroach /usr/local/bin/
-          cockroach start --insecure --background
+          wget -qO- https://binaries.cockroachdb.com/cockroach-v21.1.19.linux-amd64.tgz | tar -xz
+          mv cockroach-v21.1.19.linux-amd64/* .
+          ln -s `pwd`/cockroach /usr/local/bin/
```

Comment:
The recent version has two .so libraries so just make keep them in a single directory. However, the CLI should be in the PATH so just link it.